app/scripts/modules/core/src/pipeline/config/stages/webhook/webhookStage.ts

```diff
@@ -101,6 +109,12 @@ export class WebhookStage implements IController {
     this.stage.statusUrlResolution = this.viewState.statusUrlResolution;
   }
 
+  public failFastCodesChanged(): void {
+    const failFastCodes = this.viewState.failFastStatusCodes.split(/(?:,| |;)+/);
```

Comment:
What is that regex matching? If it's just a comma separated list we're after, it might read easier as:
```javascript
const failFastCodes = this.viewState.failFastStatusCodes.split(',').map(c => c.trim());
```
If we're trying to match on space or semicolon separated choices, we should update the help text above to indicate the flexibility in separators - but it's probably better to be consistent with the rest of the app and only allow comma-separated values here.

Reply:
I was trying to be more permissive, but you're right, better to be consistent and have the behavior match the docs. Will update to just accept commas

Reply:
I'd also be happy with `.split(/[,\s]+/)`

Reply:
I like the initiative!